examples/us-dams/package.json

```
@@ -0,0 +1,25 @@
+{
+  "type": "module",
+  "private": true,
+  "scripts": {
+    "clean": "rimraf src/.observablehq/cache",
+    "build": "rimraf dist && observable build",
+    "dev": "observable preview",
+    "deploy": "observable deploy",
+    "observable": "observable"
+  },
+  "dependencies": {
+    "@observablehq/framework": "latest",
```

```suggestion
    "@observablehq/framework": "^1.7.1",
```

If you upgrade to the latest framework, `observable create` should follow this pattern.